tracker/templates/advisory.html

```diff
@@ -17,18 +17,48 @@ <h1>{{ advisory.id }}
 				{%- endif %}
 				<a href="/{{  advisory.id }}/{% if generated %}generate/{% endif %}raw" accesskey="r">raw</a>
 			</h1>
-			<table class="styled-table full size">
-				<thead>
-					<tr>
-						<th>[{{ advisory.id }}] {{package.pkgname}}: {{advisory.advisory_type}}</th>
-					</tr>
-				</thead>
-				<tbody>
-					<tr>
-						<td>
-							<div class='advisory'>{{ raw_asa|safe|urlize }}</div>
-						</td>
-					</tr>
-				</tbody>
-			</table>
+			<div class="row">
+				<div class="one-half column">
+					<table class="styled-table full size">
+						<thead>
+							<tr>
+								<th>[{{ advisory.id }}] {{package.pkgname}}: {{advisory.advisory_type}}</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<td>
+									<div class='advisory'>{{ raw_asa|safe|urlize }}</div>
+								</td>
+							</tr>
+						</tbody>
+					</table>
+				</div>
+
+				{%- if current_user.is_authenticated and form %}
+				<div class="one-half column">
+					<table class="styled-table full size">
+						<thead>
+							<tr>
+								<th>Advisory reviews</th>
+							</tr>
+						</thead>
+						<tbody>
+							{% for review in reviews %}
+							<tr>
+								<td>{{review.user.name}} - {%- if review.approved %} <span class="green">Approved</span> {%- else %} <span class="red">Disapproved</span> {%- endif %} - {{review.note}}</td>
+							</tr>
+							{%- endfor %}
+						</tbody>
+					</table>
+
+				<form action="{{ action }}" method="post" name="add">
```

Comment:
I'm not too familiar with Flask, but I assume there's some CSRF type of protection somewhere in the form setup? Is this considered already?

Reply:
We use `WTForm` to structure the form submission and they have a handy `{{ form.hidden_tag() }}` thing we use to generate CSRF protection.